src/darwin/Framework/CHIP/MTRBaseDevice.h

```diff
@@ -315,6 +315,50 @@ typedef NS_ENUM(uint8_t, MTRTransportType) {
                                       completion:(MTRDeviceOpenCommissioningWindowHandler)completion
     API_AVAILABLE(ios(16.2), macos(13.1), watchos(9.2), tvos(16.2));
 
+/**
+ * Reads events from the device.
+ *
+ * Nil values for endpointID, clusterID, eventID indicate wildcards
+ * (e.g. nil eventID means "read all the events from the endpoint(s) and
+ * cluster(s) that match endpointID/clusterID").
+ *
+ * If all of endpointID, clusterID, eventID are non-nil, a single
+ * event will be read.
+ *
+ * If all of endpointID, clusterID, eventID are nil, all events on the
+ * device will be read.
+ */
+
+- (void)readEventsWithEndpointID:(NSNumber * _Nullable)endpointID
+                       clusterID:(NSNumber * _Nullable)clusterID
+                         eventID:(NSNumber * _Nullable)eventID
+                        eventMin:(NSNumber * _Nullable)eventMin
```

Comment:
I'd argue eventMin should be in MTRReadParams, not a separate argument here.  And it should document what the behavior is if it's left nil.

Reply:
eventMin is into MTRReadParams and documented based on Matter 1.0 Specification.